plugins/out_azure_kusto/azure_kusto.c

```diff
@@ -1034,7 +1035,31 @@ static int azure_kusto_format(struct flb_azure_kusto *ctx, const char *tag, int
         msgpack_pack_str(&mp_pck, flb_sds_len(ctx->log_key));
         msgpack_pack_str_body(&mp_pck, ctx->log_key, flb_sds_len(ctx->log_key));
 
-        if (log_event.body != NULL) {
+        if (log_event.group_attributes != NULL && log_event.body != NULL) {
+            msgpack_pack_map(&mp_pck,
+                                 log_event.group_attributes->via.map.size +
+                                 log_event.metadata->via.map.size +
+                                 log_event.body->via.map.size);
+
+            for (index = 0; index < log_event.group_attributes->via.map.size; index++)
```

Comment:
braces in `for` opens in the right: https://github.com/fluent/fluent-bit/blob/master/CONTRIBUTING.md#braces-usage-on-conditionals-loops-and-functions